main.py

```diff
@@ -160,6 +167,21 @@ def scrape(self, site, depth=0):
             content_type = self.get_content_type(headers)
             print("link", href, content_type)
             # if this is a document type, store it
+
+            if new:
+                if self.total_new_gdoc_count >= MAX_NEW_GOOGLE_DOCS:
+                        break
+                if GDRIVE_URL in full_href:
+                    self.set_message(f"Processing Google Drive URL: {full_href}")
+                    try:
+                        if grab(href, "./out"):
+                            self.set_message(f"Captured Google Drive file: {full_href}")
+                            self.total_new_gdoc_count += 1
+                    except: 
+                        # If there is gdrive site that fails to download, we can remove it from the seen list and move on. 
```

Comment:
We do not want to remove links from the seen list - it is used to not re-download the same file next run.  If we remove links that don't work, we will retry them on every scrape.